### PR TITLE
Parser for web menicka.cz and some restaurans added

### PR DESCRIPTION
**Pull request by lukasmarmol**
Parser for menicka.cz added. 
"U Kristyna" restaurant using menicka.cz parser added.
"Na Knofliku" and "Goool" restaurants redefined to use menicka.cz parser.

### CODE REVIEW

File: LunchTime/Restaurants/MenickaCz/NaKnofliku.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+{
+    public class NaKnofliku : MenickaCzBase
+    {
+        public override string Name => "Na Knoflíku";
+
+        public override string Url => "https://www.menicka.cz/3899-na-knofliku.html";
+
+        public override string Web => "";
+
+        public override GeoCoordinate Location => new GeoCoordinate(0, 0);
```
Comment (martinsamanek) [inline LunchTime/Restaurants/MenickaCz/NaKnofliku.cs]: Missing coordinates. They were in previous implementation

File: LunchTime/Restaurants/MenickaCz/NaKnofliku.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+namespace LunchTime.Restaurants.MenickaCz
+{
+    public class NaKnofliku : MenickaCzBase
+    {
+        public override string Name => "Na Knoflíku";
+
+        public override string Url => "https://www.menicka.cz/3899-na-knofliku.html";
+
+        public override string Web => "";
```
Comment (martinsamanek) [inline LunchTime/Restaurants/MenickaCz/NaKnofliku.cs:16]: Missing coordinates. It was in the previous implementation

File: LunchTime/Restaurants/MenickaCz/UKristyna.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+{
+    public class UKristyna : MenickaCzBase
+    {
+        public override string Name => "U Kristýna";
+
+        public override string Url => "https://www.menicka.cz/5471-u-kristyna.html";
+
+        public override string Web => "";
+
+        public override GeoCoordinate Location => new GeoCoordinate(0, 0);
```
Comment (martinsamanek) [inline LunchTime/Restaurants/MenickaCz/UKristyna.cs]: Missing coordinates

**Review by martinsamanek — CHANGES_REQUESTED** [review]

File: LunchTime/Restaurants/MenickaCz/UKristyna.cs (hunk 3 above)
Comment (lukasmarmol) [reply LunchTime/Restaurants/MenickaCz/UKristyna.cs]: fixed - coords added from mapy.cz

**Pull request merged**